pkg/translator/prometheusremotewrite/go.mod

```diff
@@ -0,0 +1,20 @@
+module github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator/prometheusremotewrite
```

Comment:
Is there anything specific to remotewrite here as opposed to just calling it `prometheus`?

Reply:
Not really. I can change it if there is consensus for `prometheus`

Reply:
This is now done

Reply:
Isn't this all specific to remote write?  This would not convert to the prometheus or openmetrics text exposition formats.  These types are only used for remote write.